src/lib.rs

```diff
@@ -377,7 +402,39 @@ impl RenderPass {
             self.user_textures.push(Some(bind_group));
         }
     }
+    /// replace given texture id 's buffer to given wgpu:texture
+    /// copy by gpu so data is in device memory
+    /// Should be called before `execute()`.
+    pub fn over_write_texture(&mut self,device:&wgpu::Device,
```

Comment:
update_texture() ?

Reply:
Maybe this function is not necessary and we should implement a function like texture_from_texture_id instead. It's much simpler and easier to understand.